README.md

```diff
@@ -300,7 +301,7 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - :video_camera: [Java + DSA + Interview Preperation Playlist](https://www.youtube.com/playlist?list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ)
 
 ### C++
-
+- :video_camera: [Love babbar dsa playlist](https://youtube.com/playlist?list=PLDzeHZWIZsTryvtXdMr6rPh4IDexB5NIA&si=ciHWfoFmDDcNwE2k)
```

Comment:
Keep a empty line between heading and content...

Reply:
done! can you check again